lib/components/form/settings-preview.js

```diff
@@ -22,11 +23,16 @@ class SettingsPreview extends Component {
   }
 
   static defaultProps = {
-    editButtonText: <i className='fa fa-pencil' />
+    editButtonText: <i className='fa fa-pencil' />,
+    messages: {
+      // Label accepted as two-element array or single (not array) string literal.
```

Comment:
What happens if someone passes `messages` with attributes that don't include `label`?

Reply:
`mergeMessages` will ensure that the default label key gets preserved.

Reply:
Ok the answer was below in `mergeMessages`.